articles/aks/advanced-network-observability-cli.md

```diff
@@ -250,7 +250,7 @@ Install the Hubble CLI to access the data it collects using the following comman
 
 ```azurecli-interactive
 # Set environment variables
-export HUBBLE_VERSION=v0.11.0
+export HUBBLE_VERSION=$(curl -s https://raw.githubusercontent.com/cilium/hubble/master/stable.txt)
```

Comment:
I wanted to keep it fixed on a version to make sure we have tested it and it works as expected. Pulling the latest version doesn't gurantee that.

Reply:
Thanks Anubhab, I will test on the latest version v1.16.2 and if all is in order I can hardcode to that.